uaclient/security.py

```diff
@@ -813,6 +815,8 @@ def _handle_released_package_fixes(
                     if not binary_pkgs:
                         print(status.MESSAGE_SECURITY_UPDATE_INSTALLED)
                         continue
+                    else:
+                        all_already_installed = False
```

Comment:
I don't think we can just set this variable to `False` here directly. We are looking at three distinct upgrade pockets in this function, `standard updates, esm-apps and esm-infra`. Maybe one of these pockets do not have binary packages to install, but the other do

Reply:
Yes I agree! Note though that this boolean is supposed to only be True when all the binary packages that fix the CVE/USN were already installed.
I'm setting this to False here because in this case, there _are_ packages to be installed, which means that we're not in the special case where they were _all_ already installed.
If we reach this case for at least one of the pockets, then we can't say that "packages were already installed" and the new special case added below won't apply.

Reply:
That's fair, but I think that if at least one package was installed, it could require a reboot. That's why I think we should block the reboot message only if no packages were installed at all